library/std/src/sys/fs/windows.rs

```diff
@@ -1057,6 +1057,8 @@ impl FileAttr {
     }
 }
 impl From<c::WIN32_FIND_DATAW> for FileAttr {
+    /// Map `c::WIN32_FIND_DATAW`s fields to a `FileAttr`,
+    /// with some conditionals for `file_size` and `reparse_tag`.
```

Comment:
"with some conditionals" is not very helpful: this documentation neither informs me more than simply reading the signature, nor does it spare me from reading the source code